WordPress/src/main/res/raw/mocked_posts_data.json

```diff
@@ -4,35 +4,40 @@
     "draft": [
       {
         "id": 3172,
-        "title": "A draft post 1...",
+        "title": "Vegetable salad",
+        "excerpt": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
         "modified": "2021-07-27T16:07:45+00:00",
-        "featured_image_url": "https://draft_1_feature_image_url"
+        "featured_image_url": "https://images.pexels.com/photos/1640777/pexels-photo-1640777.jpeg?h=150"
```

Comment:
1. Suggestion (💡): Let's have one `featured_image_url` as `null` for the drafts (this first draft) and one more for the scheduled (say the second scheduled), then we will win this extra visual combination during testing.
2. Suggestion (💡): Let's also have one `title` as `null` for, say, the first scheduled and win this extra visual combination during testing as well.

Reply:
In 59eeccd

I also included a long title and updated the screenshot in the PR description.

Reply:
Great, thank Youuuuuuuuu!